tests/acceptance/features/webUITrashbin/trashbinDelete.feature

```diff
@@ -47,6 +47,7 @@ Feature: files and folders can be deleted from the trashbin
 
   @skip @yetToImplement
   Scenario: Select all except for some files and delete from trashbin in a batch
+    # no batch delete in phoenix yet
```

Comment:
why is it not possible here but in the next step?
does deleting all files work but not some?

Reply:
We have a deleted all Button in trashbin but Not in the Filelist.

It is Missing as of now 🤷